### PR TITLE
Oleksii/ob 294 create zip file

### PR DESCRIPTION
**Pull request by alexone55**

### CODE REVIEW

File: Alexey/Files/create_zip_file/tests/test_create_zip_file.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import unittest
+import os
+from Alexey.Files.create_zip_file.main.create_zip_file import all_files_in_folder, archive_maker
+
+
+class TestCreateZipFileMaker(unittest.TestCase):
```
Comment (glock3) [inline Alexey/Files/create_zip_file/tests/test_create_zip_file.py:6]: Add tests for `archive_maker` function

File: Alexey/Files/create_zip_file/tests/test_create_zip_file.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        expect_set = ['NewZip.zip', 'test_create_zip_file.py', 'test_file.txt']
+        actual_set = all_files_in_folder()
+        self.assertEqual(expect_set, actual_set)
+
+    def test_make_new_zip_file(self):
+        expect_set = ['NewZip.zip']
+        archive_maker(all_files_in_folder())
+        actual_set = []
+        files_set = all_files_in_folder()
+        for file in files_set:
```
Comment (glock3) [inline Alexey/Files/create_zip_file/tests/test_create_zip_file.py]: `all_files_in_folder()` already returns a list, why do you need this `for` loop?

File: Alexey/Files/create_zip_file/tests/test_create_zip_file.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            if file == 'NewZip.zip':
```
Comment (glock3) [inline Alexey/Files/create_zip_file/tests/test_create_zip_file.py]: I don't think that filtering actual_set is a good approach

File: Alexey/Files/create_zip_file/tests/test_create_zip_file.py (hunk 1 above)
Comment (glock3) [inline Alexey/Files/create_zip_file/tests/test_create_zip_file.py:6]: Also add tests for empty file.

**Review by glock3 — CHANGES_REQUESTED** [review]

File: Alexey/Files/create_zip_file/tests/test_create_zip_file.py (hunk 2 above)
Comment (alexone55) [reply Alexey/Files/create_zip_file/tests/test_create_zip_file.py]: done

File: Alexey/Files/create_zip_file/tests/test_create_zip_file.py (hunk 3 above)
Comment (alexone55) [reply Alexey/Files/create_zip_file/tests/test_create_zip_file.py]: done

File: Alexey/Files/create_zip_file/tests/test_create_zip_file.py (hunk 1 above)
Comment (alexone55) [reply Alexey/Files/create_zip_file/tests/test_create_zip_file.py:6]: done

Comment (alexone55) [reply Alexey/Files/create_zip_file/tests/test_create_zip_file.py:6]: done

**Review by glock3 — APPROVED** [review]
